### PR TITLE
feat: add logout button

### PR DESCRIPTION
**Pull request by pea-nut-z**

### CODE REVIEW

File: app/src/components/LogoutSpotifyButton.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    const logout = async () => {
+        try {
+            await signUserOut()
+        } catch (error) {
+        console.error("Logout Error: ",error);
+        }
+    };
+    
+  return (
+    <button
```
Comment (0916dhkim) [inline app/src/components/LogoutSpotifyButton.tsx:17]: Let's replace this `button` tag with the `Button` component. You can find it in `app/src/components/Button.tsx`.
(I should update the `ConnectSpotifyButton` to use `Button` as well).

File: app/src/firebase.tsx [hunk 2]
```diff
@@ -51,6 +52,10 @@ export const signIn = async (email: string, password: string) => {
   return signInWithEmailAndPassword(auth, email, password);
 };
 
+export const signUserOut = () => {
+  return signOut(auth);
+};
```
Comment (0916dhkim) [inline app/src/firebase.tsx:57]: :100:

File: app/src/components/LogoutSpotifyButton.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { ReactElement } from "react";
+import { button } from "./ConnectSpotifyButton.css";
```
Comment (0916dhkim) [inline app/src/components/LogoutSpotifyButton.tsx:2]: `ConnectSpotifyButton.css` will not be necessary if we use the `Button` component.

**Review by 0916dhkim — COMMENTED** [review]
Thanks for opening a PR! Just a couple modifications and I can merge the PR :+1:

**Comment by pea-nut-z** [comment]
Fixed. Please take a look.

File: app/src/components/LogoutSpotifyButton.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            await signUserOut()
+        } catch (error) {
+        console.error("Logout Error: ",error);
+        }
+    };
+    
+  return (
+    <Button
+    variant="primary"
+    className={sprinkles({ minWidth: { mobile: "1/2", tablet: "1/4" } })}
```
Comment (0916dhkim) [inline app/src/components/LogoutSpotifyButton.tsx:21]: :+1:

**Review by 0916dhkim — APPROVED** [review]
Looks good to me! Merging.

**Pull request merged**